_posts/2016/04/2016-04-20-tips-tricks-live-coding.md

```diff
@@ -0,0 +1,73 @@
+
+Regardless of whether you are standing or sitting, make sure to move around occasionally. You can for example move to the screen to point something out, or draw something on the white/blackboard (see below). Moving around makes the teaching more lively, less monotonous. It draws the learners' attention away from their screens, to you, which helps getting the point you are making across.
+
+Even though you may have a good voice and know how to use it well, it may be an idea to use a microphone, especially if the workshop room is equipped with one. Your voice will be less tired, and you increase the chance of people with hearing difficulties being able to follow the workshop.
+
+### 2. Take it slow
+
+For every command you type, every word of code you write, every menu item or website button you click, say out loud what you are doing while you do it. This not only slows you down, it allows learners who are following along to copy what you do, even when they are looking at their screen while doing it. If the output of your command or code makes what you just typed disappear from view, scroll back up so learners can see it again - this is especially needed for the unix shell lesson.
+
+Other possibilities are to execute the same command a second time, or copy and paste the last command(s) into the workshop Etherpad.
+
+### 3. Mirror your learner's environment as much as possible
+
+You may have set up your environment to your liking, with a very simple or rather fancy unix prompt, colour schemes for your development environment, keyboard shortcuts etc. Your learners usually won't have all of this. Try to create an environment that mirrors what your learners have, and avoid using keyboard shortcuts. Some instructors create a separate 'bare-bone' user (login) account on their laptop, or a separate 'teaching-only' account on the service being taught (e.g. github).
+
+### 4. Use the screen - or screens - wisely
+
+Use a big font, and maximise the window. A black font on a  white background works better than a light font on a dark background. When the bottom of the projector screen is at the same height, or below, the heads of the learners, people in the back won't be able to see the lower parts. Draw up the bottom of your window(s) to compensate. 
+
+If you can get a second screen, use it! It will usually require its own PC or laptop, so you may need to ask a helper to control it. You could use the second screen to show the Etherpad content, or the lesson material, or illustrations.
+
+Pay attention to the lightning (not too dark, no lights directly on/above the presenter's screen) and if needed, reposition the tables so all learners can see the screen, and helpers can easily reach all learners.
```

Comment:
Yes, viking thing. Something with the hammer (SWC logo), Thor and before you know it, you have lightning. Good to pay attention to should it happen at your workshop...